Sources/TuistKit/Graph/GraphLoader.swift

```diff
@@ -61,7 +61,7 @@ class GraphLoader: GraphLoading {
     fileprivate func loadWorkspace(path: AbsolutePath) throws -> Graph {
         let cache = GraphLoaderCache()
         let circularDetector = GraphCircularDetector()
-        let workspace = try Workspace.at(path)
+        let workspace = try modelLoader.loadWorkspace(at: path)
```

Comment:
I was debating keeping it consistent with `Project.at`:

```swift
static func at(_ path: AbsolutePath, modelLoader: GeneratorModelLoading) throws -> Workspace {
    return try modelLoader.loadWorkspace(at: path)
}
```

Seeing it would have just called the `modelLoader` -  opted to so directly.